### PR TITLE
Generate info DBs in shelve format

### PR DESCRIPTION
**Pull request by morsecodist**
Generate a shelve file for the info db as well.

## Test plan

I will run this on the 10-01 configs then do some custom pipeline runs with that config and manually setting the path. In theory it should work with no changes to the dag code itself.

### CODE REVIEW

File: examples/generate_coverage_viz.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
       "class": "PipelineStepGenerateCoverageViz",
       "module": "idseq_dag.steps.generate_coverage_viz",
       "additional_files": {
-        "info_db": "s3://idseq-database/alignment_data/2018-12-01/nt_info.sqlite3"
+        "info_db": "s3://idseq-database/alignment_data/2018-12-01/nt_info.db"
```
Comment (gregdingle) [inline examples/generate_coverage_viz.json]: why replace? in other steps, we are uploading both sqlite and db files

File: idseq_dag/steps/generate_loc_db.py [hunk 2]
```diff
@@ -85,32 +84,48 @@ def generate_loc_db_for_sqlite(self, db_file, loc_db_file, info_db_file):
...
+
                     seq_offset = seq_offset + header_len + seq_len
                     header_len = len(line)
                     seq_len = 0
-                    s = re.match('^>([^ ]*).*', line)
+                    seq_bp_len = 0
+                    accession_name = ""
+                    # Sometimes multiple accessions will be mapped to a single sequence.
+                    # In this case, they will be separated by the \x01 char.
+                    # To get the accession name, just match until the first \x01.
```
Comment (gregdingle) [inline idseq_dag/steps/generate_loc_db.py]: so are we preferring the first accession then? by what justification?

Comment (gregdingle) [reply idseq_dag/steps/generate_loc_db.py]: also it seems like this is copy pasted without comment from a method above

**Comment by morsecodist** [comment]
@gregdingle @boris-dimitrov is interested in eliminating the sqlite dependency from runtime. Previously I had a task to convert 100% to shelve but I missed this conversion because it never had a shelve version. So a bug was raised on getting this done and was assigned to me.

File: examples/generate_coverage_viz.json (hunk 1 above)
Comment (morsecodist) [reply examples/generate_coverage_viz.json]: See comment about eliminating sqlite support at runtime the long term.

File: idseq_dag/steps/generate_loc_db.py (hunk 2 above)
Comment (morsecodist) [reply idseq_dag/steps/generate_loc_db.py]: Your second comment is correct, I will add a comment, though it is not 100% copy pasted. The logic should be identical to the sqlite logic which is why I made this transfer. I am not sure why we made this choice, this is only concerning the format of the resulting key value store. I will add a comment.

**Comment by morsecodist** [comment]
@gregdingle could you expand a bit on what you mean by?

> make sure that this info db generation is consistent with the PipelineStepGenerateLocDB code

**Comment by MarkAZhang** [comment]
Thanks for doing this!

**Pull request merged**